docs/content/preview/yugabyte-platform/alerts-monitoring/anywhere-export-configuration.md

```diff
@@ -0,0 +1,129 @@
+
+Currently, you can export data to the following tools:
+
+- [Datadog](https://docs.datadoghq.com/)
+- [Splunk](https://www.splunk.com/en_us/solutions/opentelemetry.html)
+- [AWS CloudWatch](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/WhatIsCloudWatch.html)
+- [GCP Cloud Logging](https://cloud.google.com/logging/)
+
+For information on how to export logs from a universe using an export configuration, refer to [Export logs](../universe-logging/).
+
+## Prerequisites
+
+Export configuration is {{<badge/ea>}}. To enable export configuration management, set the **Enable DB Audit Logging** Global Configuration option (config key `yb.universe.audit_logging_enabled`) to true. Refer to [Manage runtime configuration settings](../../administer-yugabyte-platform/manage-runtime-config/). Note that only a Super Admin user can modify Global configuration settings. The flag can't be turned off if audit logging is enabled on a universe.
+
+## Configure integrations
+
+Create and manage export configurations on the **Integrations > Log** page.
+
+![Export configurations](/images/yp/export-configurations.png)
+
+The page lists the configured and available third-party integrations.
+
+### Manage integrations
+
+You can add and delete export configurations for the following tools. You can't delete a configuration that is in use by a universe.
+
+{{< tabpane text=true >}}
+
+  {{% tab header="Datadog" lang="datadog" %}}
+
+The Datadog export configuration requires the following:
+
+- Datadog account
+- Datadog [API key](https://docs.datadoghq.com/account_management/api-app-keys/)
+
+To create an export configuration, do the following:
+
+1. On the **Integrations** page, on the **Log** tab, click **Create Export Configuration**, and choose **Datadog**.
+1. Enter a name for the configuration.
+1. Enter your Datadog [API key](https://docs.datadoghq.com/account_management/api-app-keys/).
+1. Choose the Datadog site to connect to, or choose Self-hosted and enter your URL.
+1. Click **Create Configuration**.
+
+  {{% /tab %}}
+
+  {{% tab header="Splunk" lang="splunk" %}}
+
+The Splunk export configuration requires the following:
+
+- Splunk access token
+- Endpoint URL
+
+To create an export configuration, do the following:
+
+1. On the **Integrations** page, on the **Log** tab, click **Create Export Configuration**, and choose **Splunk**.
+1. Enter a name for the configuration.
+1. Enter your Splunk [Access token](https://docs.splunk.com/observability/en/admin/authentication/authentication-tokens/org-tokens.html).
+1. Enter the Endpoint URL.
+1. Optionally, enter the Source, Source Type, and Index.
+1. Click **Validate and Create Configuration**.
+
+  {{% /tab %}}
+
+  {{% tab header="AWS" lang="aws" %}}
+
+The AWS CloudWatch export configuration requires the following:
+
+- Access Key ID and Secret Access Key for the AWS Service Account
```

Comment:
Should we mention the other requirements such as Log group, Log stream, and region here?